### DISCUSSION TITLE
How u getting item data when cursor is on a item ?

### DISCUSSION THREAD

**Discussion by YKarayel** [SnosMe/awakened-poe-trade#1369] [Q&A]
Hello there :) When we users press the ctrl +d keys while the mouse cursor is on the item, we get information about the item. But i wonder how u guys getting this data from game ?

I am very curious about the answer to this question. I would be very happy if you answer it. Thanks ^^

**Comment by SnosMe** [comment]
emulate key press, wait for item to appear in clipboard.

![image](https://github.com/SnosMe/awakened-poe-trade/assets/4292308/6cb749c2-43fa-4766-aee3-5b08ccdc455c)
